### ISSUE TITLE
Optional Plotting

### ISSUE THREAD

**Issue by sivasubramanics** [jiabowang/GAPIT#80]
Dear Team,
Thank you so much for the great tool!

I was looking for any particular way/option to flag out the plotting functions. What if I do not want any plots to be generated and only the csv outputs I need, do we have any option for that. If we are working on multiple scaffolds, this option might help a lot.

Best,
Siva

**Comment by jiabowang** [comment]
Hi Siva,

Thanks for your suggestion.
We are working on that function in the GAPIT you wanted.
But now it does not work.
All figures are only output by automatic after analysis.

**Comment by liuye2024** [comment]
Dear Wang,
May I ask if the function that only outputs analysis results without drawing plots can be used? When I was using the farmcpu model for analysis on the cluster, I encountered an "out of memory" issue in the plotting section, which resulted in the SNP P-values not being output. Looking forward to your reply. Thank you very much.
![farmcpuerror](https://github.com/jiabowang/GAPIT/assets/155537889/7423153b-c7d2-4cbe-9c7e-c44ee2fa54f9)

**Comment by jiabowang** [comment]
Hi,
We can use output.file=FALSE not to output any analysis results files. But from your report, I guess the computing memory caused that. Please divide genotype file into different chromosome, that should work for your analysis.

**Comment by liuye2024** [comment]
Dear Wang,
Thank you for your response. Will the command output.file=FALSE still generate a file containing SNP p-values? Thank you.

**Comment by jiabowang** [comment]
@liuye2024 No, all results will not be generated. Did you use FarmCPU software, or the GAPIT?

**Comment by liuye2024** [comment]
I haven't used the FarmCPU software mentioned in the link "https://github.com/amkusmec/FarmCPUpp." I have only used GAPIT3 before, so I think I will give GAPIT a try. As for the BLINK_C software, if there are no significant loci and no output files, I am inclined to choose the GAPIT3 R package over the FarmCPU or BLINK models.Thanks a lot.
